internal/flowrate/io_test.go

```diff
@@ -198,3 +233,22 @@ func ratesAreEqual(r1 int64, r2 int64) bool {
 	}
 	return false
 }
+
+func statusesAreEqualWithinTolerance(s1 *Status, s2 *Status, maxDeviationForDuration time.Duration, maxDeviationForRate int64) bool {
```

Comment:
Can't we modify `statusesAreEqual`? Having two functions that do the same thing seems excessive.

Reply:
Think so?  I'll check on my flight.

Reply:
hey, it seems my response did not post -- the answer to your question is"

- "if you are oaky with all the things being statusesAeEqualWithinTolerance" then yep, we can do that.  Currently equal means exactly equal and within tolerance gives them some wiggle room.